### PR TITLE
[2/3 dedupe] Remove suspend modal dupe

### PR DESCRIPTION
**Pull request by rcorral**

### CODE REVIEW

**Comment by mlunoe** [comment]
Rebased for you

File: src/js/components/modals/ServiceSuspendModal.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+import MarathonStore from '../../stores/MarathonStore';
+import ServiceTree from '../../structs/ServiceTree';
+import ServiceActionModal from './ServiceActionModal';
+
+class ServiceSuspendModal extends ServiceActionModal {
+  constructor() {
```
Comment (mlunoe) [inline src/js/components/modals/ServiceSuspendModal.js]: `...arguments`

**Comment by mlunoe** [comment]
Great work! 👍 💯 :shipit:

**Comment by orlandohohmeier** [comment]
LGTM 💯 ✨

**Pull request merged**